feature/author/src/main/java/com/google/samples/apps/nowinandroid/feature/author/AuthorScreen.kt

```diff
@@ -91,6 +93,7 @@ internal fun AuthorScreen(
     onBookmarkChanged: (String, Boolean) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    TrackScrollJank(scrollableState = rememberLazyListState(), stateName = "author:column")
```

Comment:
Should the `LazyListState` passed to `TrackScrollJank` also be passed to the `LazyColumn`?